webtool/test_correct.py

```
@@ -1812,3 +1812,268 @@ def test_cmd_run_ueberlebt_ein_nicht_versucht(monkeypatch, tmp_path):
+
+    # Vorbedingung fuer den Loeschzweig: das Sidecar ist AELTER als die Roh-JSON, der
+    # Wiederverwendungs-Skip greift also nicht (`mtime(dpath) >= mtime(raw_json)`).
+    dpath = tdir / "S1.diar.json"
+    dpath.write_text(json.dumps({"base": "S1", "turns": [], "segments": []}), encoding="utf-8")
+    alt = os.path.getmtime(tdir / "S1.json") - 60
+    os.utime(dpath, (alt, alt))
+
+    echt_remove = os.remove
+    marke = "marke: os.remove(S1.diar.json)"
+
+    def remove_mit_marke(pfad, *a, **kw):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `remove_mit_marke`

(ANN202)